client/site-profiler/components/metrics-insights-list/index.tsx

```diff
@@ -18,9 +19,15 @@ const Container = styled.div`
 	letter-spacing: -0.1px;
 `;
 
+type InsightHeaderProps = {
+	locked: boolean;
+	children: React.ReactNode;
+};
 const InsightHeader = styled.div`
 	font-family: 'SF Pro Text';
 	font-size: 16px;
+	filter: ${ ( props: InsightHeaderProps ) => ( props.locked ? 'blur(2px)' : 'none' ) };
```

Comment:
Using the `blur` filter works as expected. However, I could easily see the hidden text by toggling one CSS rule. Is there any better way to achieve similar results? Do we want to obfuscate the code in any way?

Reply:
My idea here is really to use text that doesn't mean much to be blurred (as the sentences I added in `useLockedInsights`). 

This should also help us to not need to retrieve any information at this step, which will not be available yet.

Reply:
I think if we want to obfuscate the text we could generate a temp image from the backend with the blurred text.

Reply:
But at this point, we don't have the data, the blurred text is to be shown during teaser stage.